--- dust3d/dust3d.pro	2020-04-07 10:42:08.246213390 -0400
+++ dust3d/dust3d.pro.flatpak	2020-04-14 13:20:11.609396245 -0400
@@ -881,14 +881,14 @@
 	GMP_LIBNAME = gmp
 	MPFR_LIBNAME = mpfr
 	CGAL_LIBNAME = CGAL
-	BOOST_INCLUDEDIR = /usr/local/include
-	CGAL_INCLUDEDIR = /usr/local/include
-	CGAL_BUILDINCLUDEDIR = /usr/local/include
-	CGAL_LIBDIR = /usr/local/lib
-	GMP_INCLUDEDIR = /usr/local/include
-	GMP_LIBDIR = /usr/local/lib
-	MPFR_INCLUDEDIR = /usr/local/include
-	MPFR_LIBDIR = /usr/local/lib
+	BOOST_INCLUDEDIR = /app/include
+	CGAL_INCLUDEDIR = /app/include
+	CGAL_BUILDINCLUDEDIR = /app/include
+	CGAL_LIBDIR = /app/lib
+	GMP_INCLUDEDIR = /app/include
+	GMP_LIBDIR = /app/lib
+	MPFR_INCLUDEDIR = /app/include
+	MPFR_LIBDIR = /app/lib
 }
 
 INCLUDEPATH += $$BOOST_INCLUDEDIR
